### PR TITLE
Add form validation using Javascript

### PR DESCRIPTION
**Pull request by c00p75**
Added JavaScript validation to email address input in contact section.

### CODE REVIEW

File: Script.js [hunk 1]
```diff
@@ -215,4 +215,36 @@ workSection.addEventListener('click', (event) => {
...
+    setError(email, 'Invalid email address');
+  } else {
+    setSuccess(email);
+  }
+}
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  validateEmail();
+});
```
Comment (1GORDON) [inline Script.js]: Great job @c00p75 🤝

- [ ] Kindly make sure the form submits when the email is in lowercase and also display a message if the email is in Uppercase. Currently, the form can't submit.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @c00p75 ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions

- [ ] Kindly customize your readme file again and remove irrelevant sections as far as this project is concerned. Ie the Presentation video link.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Script.js [hunk 2]
```diff
@@ -215,4 +215,39 @@ workSection.addEventListener('click', (event) => {
...
+  submitButton.value = 'Sent';
+  submitButton.classList.add('success');
+}
+
+function validateEmail() {
+  const emailValue = email.value.trim();
+  if (!emailRegex.test(emailValue)) {
+    setError(email, 'Invalid email address');
+  } else {
+    setSuccess(email);
```
Comment (Munsa1) [inline Script.js]: - [ ] The email is being submitted even if it has an uppercase value, kindly fix this issue

image

image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @c00p75 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Script.js [hunk 3]
```diff
@@ -215,4 +215,49 @@ workSection.addEventListener('click', (event) => {
...
+  submitButton.classList.add('success');
+  submitButton.disabled = true;
+}
+
+function validateEmail() {
+  if (email.value !== email.value.toLowerCase()) {
+    setError(email, 'Invalid email address');
+  } else {
+    setSuccess(email);
+  }
```
Comment (christianbiring1) [inline Script.js:249]: - [ ] Kindly make sure your email input is validated correctly.
`Hint: After checking if the email is not equal to its lowercase, you may consider preventing the form to be submitted`
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @c00p75 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
# STATUS: REQUEST CHANGES :recycle: 
163947610-e4d849d5-e69c-49b9-90bd-a4191f4e2856


 ## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary` :+1: 
- [ ] Good commit history :+1: 

 ## Required Changes ♻️
- [ ] `AS SAID BY THE PREVIOUS REVIEWER KINDLY MAKE SURE YOUR FROM IS VALIDATED CORRECTLY`
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Script.js (hunk 3 above)
Comment (Abenezer-Tilahun) [reply Script.js:249]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @c00p75 

Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github-Flow.
✔️ Professional README  
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**